src/main/java/org/digma/intellij/plugin/assets/AssetsIndexTemplateBuilder.java

```diff
@@ -53,6 +54,7 @@ public InputStream build(Project project) {
             var data = new HashMap<String, Object>();
             JCefTemplateUtils.addCommonEnvVariables(data);
 
+            var Curenv = PersistenceService.getInstance().getState().getCurrentEnv();
```

Comment:
Please fix to java code stype, variable name starts with lower case.
Please ask Arik how to use our shared code style.

Reply:
fixed